contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -70,6 +70,50 @@ reasonable production worthy environments:
   bypass policies that are not yet GA, but which may reasonably be enabled on a
   conformant cluster (e.g., pod security policy, non-GA scheduler annotations)
 
+### Windows & Linux Considerations
+
+As of v1.14, Windows is a stable feature, but not yet included in conformance
+testing. In preparation for this, a large number of conformance tests are
+already included in Windows testing. You can see what tests have already pass by
+looking at TestGrid for results of Windows tests running on 
+[Azure](https://testgrid.k8s.io/sig-windows#aks-engine-azure-windows-master) and 
+[GCE](https://testgrid.k8s.io/sig-windows#gce-windows-master)).
+
+Generally speaking, the goals are to:
+
+- Make sure tests that are already passing remain passing. If new OS-specific
+functionality is added, it should be in a new test.
+- Ensure that new tests covering Linux-specific functionality are tagged with `[LinuxOnly]` 
+(see: [Kinds of Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests),
+  - Give future reviewers a reference to an active issue or documentation clarifying why a test
+cannot run on Windows.
+
+The tests that are running today:
+
+- Rely only on container images that already have a multi-architecture manifest
+including Windows versions, or have been ported by SIG-Windows
+(see [kubernetes-sigs/windows-testing/images](https://github.com/kubernetes-sigs/windows-testing/tree/master/images)
+- Do not depend on any functionality is different or not available on Windows. The full list
+is available in the Windows Kubernetes docs under [api](https://kubernetes.io/docs/setup/windows/intro-windows-in-kubernetes/#api). 
+A brief summary is included here as a starting point. If the docs are insufficient
+or there are more questions, please contact #SIG-Windows on Slack to get another
+reviewer.
+
+Some of the most common differences to watch for are:
+- Most user & security context options are not supported since Windows does not
+use POSIX-style users, permissions or ACLs
+- Privileged containers and host networking modes are not supported. Containers are always isolated.
+- Only NTFS is supported. Volume mounts specifying other filesystems (ext4, xfs) or mediums (memory) are not supported
+  - Projected mounts and permission masks are not supported
+  - Mappings of individual files are not supported. This means some features that rely on it such as 
+V1.Container.terminationMessagePath or writing service entries to `/etc/hosts` also do not work.
+- Network and DNS settings must be passed through CNI. Windows does not use `/etc/resolv.conf` 
+and stores networking settings in the Windows registry instead
+- Windows treats all DNS lookups with a `.` to be FQDN, not PQDN.
```

Comment:
I think this could use some explanation or example. Plus, It is not entirely true. For example, hostnames, service names are resolvable by their names alone, and they are considered PQDNs.

Let's take the default namespace for example. A pod in that namespace will be able to resolve ``kubernetes`` and ``kubernetes.default.svc.cluster.local``:

    kubectl exec test-pod -- powershell -Command "Resolve-DnsName kubernetes"
    
    Name                                           Type   TTL   Section    IPAddres
                                                                       s
    ----                                           ----   ---   -------    --------
    kubernetes.default.svc.cluster.local           A      5     Answer     10.0.0.1

But, it will not be able to resolve anything in between:

    kubectl exec test-pod -- powershell -Command "Resolve-DnsName kubernetes.default"
    Resolve-DnsName : kubernetes.default : DNS name does not exist
    At line:1 char:1
    + Resolve-DnsName kubernetes.default
    + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        + CategoryInfo          : ResourceUnavailable: (kubernetes.default:String)
        [Resolve-DnsName], Win32Exception
        + FullyQualifiedErrorId : DNS_ERROR_RCODE_NAME_ERROR,Microsoft.DnsClient.C
       ommands.ResolveDnsName

    command terminated with exit code 1

Reply:
I don't see how the statement is not true. If it has a `.` it is treated as an FQDN, if not as a PQDN. The reason `kubernetes` works is because of the search path.

Reply:
Indeed, I misread it. I thought it meant that PQDNs are not supported at all.

Reply:
I added examples. That should be clearer now